helmfile/values/fluentd-sc-configmap.yaml.gotmpl

```diff
@@ -8,19 +9,48 @@ configMaps:
       @type prometheus
       port 24231
     </source>
-    # TCP input to receive logs from the forwarders
+
+    # input plugin that collects metrics from MonitorAgent
     <source>
-      @type forward
-      bind 0.0.0.0
-      port 24224
+      @type prometheus_monitor
+      <labels>
+        host ${hostname}
+      </labels>
+    </source>
+
+    # input plugin that collects metrics for output plugin
+    <source>
+      @type prometheus_output_monitor
+      <labels>
+        host ${hostname}
+      </labels>
+    </source>
+
+    # input plugin that collects metrics for in_tail plugin
+    <source>
+      @type prometheus_tail_monitor
+      <labels>
+        host ${hostname}
+      </labels>
     </source>
 
+    # Ignore fluentd own events
+    <match fluent.**>
+      @type null
```

Comment:
Would this rule drop all `fluentd.healthcheck` events before they are passed to stdout?